visidata/loaders/fixed_width.py

```diff
@@ -84,8 +84,7 @@ def save_fixed(vd, p, *vsheets):
             widths = {}  # Column -> width:int
             # headers
             for col in Progress(sheet.visibleCols, gerund='sizing'):
-                maxWidth = col.getMaxWidth(sheet.rows)
-                widths[col] = col.width if col.width >= maxWidth else sheet.options.default_width or maxWidth
+                widths[col] = col.getMaxWidth(sheet.rows)
```

Comment:
```suggestion
                widths[col] = col.getMaxWidth(sheet.rows)  #1849 
```

Hey @daviewales!

You are the first person to come to the repo with preferences about the fixed width saver, so I am complete for just merging in your preference.

Would you mind adding this change that includes a reference towards the issue? This is so if someone in the future comes by with a different preference, we have a reminder of the previous conversation with yours.

Thanks for submitting a PR with the behaviour you wanted!